src/main/python/systemds/matrix/operation_node.py

```diff
@@ -229,3 +233,10 @@ def moment(self, moment, weights: DAGNode = None) -> 'OperationNode':
             unnamed_inputs.append(weights)
         unnamed_inputs.append(moment)
         return OperationNode(self.sds_context, 'moment', unnamed_inputs, output_type=OutputType.DOUBLE)
+
+    def lm(self, y: DAGNode, **kwargs) -> 'OperationNode':
```

Comment:
(Suggestion) Could it make sense to move this lm function to a new file?

I can imagine that if we add all functions to this file it will become harder to look over.

Reply:
Hi, do you have any suggestion on how the file should be called?

Reply:
Something along the lines of "buildin", and if you make it then it would be great to move l2svm along with it.
But, lest do that in another PR at some later point, because then we will merge this PR shortly